app/controllers/api/v1x0/workflows_controller.rb

```diff
@@ -43,6 +43,29 @@ def destroy
         json_response({ :message => Workflow::MSG_PROTECTED_RECORD }, :forbidden)
       end
 
+      def link
+        attrs = params.require(:obj).to_unsafe_h
+        WorkflowLinkService.new(params.require(:id)).link(attrs)
+
+        head :no_content
+      end
+
+      def unlink
+        attrs = params.require(:obj).to_unsafe_h
+        WorkflowLinkService.new(params.require(:id)).unlink(attrs)
+
+        head :no_content
+      end
+
+      def resolve
+        objs = params.require(:objs).collect(&:to_unsafe_h)
+
+        # TODO: Based on :object_id inside of objs, go to its app to retrieve tag names, and
+        #  objs.merge(:tag_name => tag_name)
+        workflow_ids = WorkflowFindService.new.find(objs)
+        json_response(workflow_ids, :created)
```

Comment:
@hsong-rh we might also have to map the object_id to workflow id and workflow_name

Reply:
@mkanoor So what's the response do you prefer? Array of key pairs?

Reply:
@mkanoor Changed to return an array of response object.